packages/api/src/external/carequality/organization-conversion.ts

```diff
@@ -1,26 +0,0 @@
-import { uuidv7 } from "@metriport/core/util/uuid-v7";
-import { XCPDGateway } from "@metriport/ihe-gateway-sdk";
-import { CQOrgBasicDetails } from "./command/cq-directory/search-cq-directory";
-
-export async function cqOrgsToXCPDGateways(cqOrgs: CQOrgBasicDetails[]): Promise<XCPDGateway[]> {
```

Comment:
Moved w/o changes to https://github.com/metriport/metriport/pull/2974/files#diff-89a78675276350d64a8435988dc39d01941058d7d1d40f525938aae273593d42R79